src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+// Create a new promise (renamed to clickPromise for clarity)
+const clickPromise = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
```

Comment:
The way you've set up your promise could lead to an issue. If the user clicks on the logo after 3 seconds, the promise will already be rejected and clicking the logo will not change its state. A better approach might be to start the timer when the user clicks the logo, not when the page loads.